lib/ansible/plugins/action/yum.py

```diff
@@ -47,7 +48,15 @@ def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
         # Carry-over concept from the package action plugin
-        module = self._task.args.get('use_backend', "auto")
+        module = 'auto'
+
+        if 'use' in self._task.args and 'use_backend' in self._task.args:
+            raise AnsibleActionFail("parameters are mutually exclusive: ('use', 'use_backend')")
+
+        use_backend = self._task.args.get('use', self._task.args.get('use_backend', None))
```

Comment:
I think this should probably be `'auto'` instead of `None` ... unless I missed something (which I'm always open to the possibility that I have):

```python
use_backend = self._task.args.get('use', self._task.args.get('use_backend', 'auto'))
```

Reply:
Agreed. Done